kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/exceptions/SpinnakerServerException.java

```diff
@@ -35,25 +32,38 @@ public class SpinnakerServerException extends SpinnakerException {
    */
   private final String rawMessage;
 
+  /* RetrofitError's response body derived as map or null if response not provided in RetrofitError or custom  message has been provided*/
```

Comment:
In retrofit2 this comes from RetrofitException, so perhaps
```suggestion
  /* retrofit's response body derived as map or null if response not provided or custom message has been provided */
```